wsd/DocumentBroker.cpp

```diff
@@ -1794,6 +1794,7 @@ void DocumentBroker::handleTileRequest(const StringVector &tokens,
     }
     else
     {
+        tile.setNormalizedViewId(session->getCanonicalViewId());
```

Comment:
I wonder why this is needed when this is done on line 1763 already?

Reply:
Not needed :+1: I must have overlooked it